### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by kienscc**

### CODE REVIEW

**Review by yeunghiepdu — APPROVED** [review]
Approved

**Comment by kienscc** [comment]
how to make the CodeQL starts scan?

File: index.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
 </head>
 <body>
     <h1>Hello world!</h1>
-    <p>Update code directly</p>
```
Comment (yeunghiepdu) [inline index.html:11]: add class hidden instead of remove

**Review by yeunghiepdu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kienscc) [reply index.html:11]: We don't need it, I think

**Review by yeunghiepdu — APPROVED** [review]
Accepted

**Comment by kienscc** [comment]
General comment here?

**Review by kienscc — COMMENTED** [review]
file comment?

File: index.html [hunk 2]
```diff
@@ -8,7 +8,6 @@
 </head>
 <body>
     <h1>Hello world!</h1>
```
Comment (kienscc) [inline index.html:10]: line comment?

**Pull request merged**